Documentation/migration-issues.md

```diff
@@ -0,0 +1,16 @@
+## Getting help on migration issues
+You're using the new .NET Core tools that are MSBuild-based. You took your project.json project and ran `dotnet migrate` or migrated from Visual Studio 2017...and you maybe ran into problems. 
+
+The best way to get help is to [file an issue](https://github.com/dotnet/cli/issues/new) on this repo and we will investigate and provide help and/or fixes as part of new CLI builds. 
```

Comment:
Most people will not read past here, I fear. Ok, at least I would not read past here. Can we include the `[Migration]` note here as well?